bot/exts/utilities/bookmark.py

```diff
@@ -71,6 +71,26 @@ class Bookmark(commands.Cog):
 
     def __init__(self, bot: Bot):
         self.bot = bot
+        self.book_mark_context_menu = discord.app_commands.ContextMenu(
+            name="Bookmark",
+            callback=self._bookmark_context_menu_callback
+        )
+        self.bot.tree.add_command(self.book_mark_context_menu)
+
+    async def cog_load(self) -> None:
+        """Carry out cog asynchronous initialisation."""
+        await self.bot.tree.sync()
+
+    @staticmethod
+    def build_bookmark_embed(target_message: discord.Message) -> discord.Embed:
+        """Build the channel embed to the bookmark requester."""
+        return discord.Embed(
+            description=(
+                f"Click the button to be sent your very own bookmark to "
+                f"[this message]({target_message.jump_url})."
+            ),
+            colour=Colours.soft_green,
+        )
```

Comment:
For consistency with the other build_* methods
```suggestion
        embed = discord.Embed(
            description=(
                f"Click the button to be sent your very own bookmark to "
                f"[this message]({target_message.jump_url})."
            ),
            colour=Colours.soft_green,
        )
        return embed
```

Reply:
Done in dc5fa83c8720a8ea90867705b1b5463b0511d40b

Reply:
I don't understand this suggestion. Some of the other build functions save embed to a param, since they need to add authors or thumbnails etc. However, this and `build_error_embed` don't, so just return the embed directly.